empress/clumpr/recon_vis/utils.py

```diff
@@ -22,55 +22,11 @@ def convert_to_objects(host_dict, parasite_dict, recon_dict):
     :return - corresondoing host_tree and parasite_tree Tree objects
         and recon Reconciliation object
     """
-    host_tree, parasite_tree = build_trees_with_temporal_order(host_dict, parasite_dict, recon_dict)
+    host_tree, parasite_tree, ok = build_trees_with_temporal_order(host_dict, parasite_dict, recon_dict)
```

Comment:
Here's what we need to do...  Rose will modify the temporal ordering code so that if the current (strict) temporal ordering isn't valid, she will try to find a second (weak) temporal ordering.  Then, the build_trees_with_temporal order should return an indication if the ordering was strict, weak, or no ordering.  Strict and weak are both fine for rendering.  If there was no ordering, the gui/cli will indicate that.

Reply:
Currently, it seems that Rose's code responds `True` when there is a strict ordering and `False` if there is no strict ordering (and set the host and parasite as none). Do you think the current code should respond to that signal (here it is indicated as ok)?